troposphere/static/js/components/modals/nullProject/NullProjectMigrateResourceModal.react.js

```diff
@@ -33,16 +34,10 @@ define(
       getInitialState: function () {
         var initialState = {
           projectName: "",
-          projects: stores.ProjectStore.getAll(),
-          projectId: null
+          projects: null,
+          projectId: -999
```

Comment:
This was done as a patchfix until we use a better base store.. I'm using -999 to mean null or not loaded, but having an empty list also evaluates to null, as a result I set that special case to -1..